h2o-core/testMultiNode.sh

```diff
@@ -76,7 +76,8 @@ else
     COVERAGE=""
 fi
 # Command to invoke test.
-JVM="nice $JAVA_CMD $COVERAGE -Xmx${MAX_MEM} -Xms${MAX_MEM} -ea -cp ${JVM_CLASSPATH} -Dsys.ai.h2o.rapids.checkObjectConsistency=true ${ADDITIONAL_TEST_JVM_OPTS}"
+# note: we set sys.ai.h2o.activeProcessorCount=4 because multinode tests are running on the same instance - this helps avoid resource contention and speeds tests up  
+JVM="nice $JAVA_CMD $COVERAGE -Xmx${MAX_MEM} -Xms${MAX_MEM} -ea -cp ${JVM_CLASSPATH} -Dsys.ai.h2o.rapids.checkObjectConsistency=true -Dsys.ai.h2o.activeProcessorCount=4 ${ADDITIONAL_TEST_JVM_OPTS}"
```

Comment:
Shouldn't the number of gc threads be also set?

Reply:
In theory, it is a good idea, however, it is currently really not needed as GC is not an issue in tests and it could cause issues with different GC implementations in different java versions that we use in tests (although currently both parallel gc and g1 would support it).